lib/tls.js

```diff
@@ -22,9 +22,9 @@ exports.DEFAULT_CIPHERS =
 
 exports.DEFAULT_ECDH_CURVE = 'prime256v1';
 
-exports.getCiphers = internalUtil.cachedResult(() => {
-  return internalUtil.filterDuplicateStrings(binding.getSSLCiphers(), true);
-});
+exports.getCiphers = internalUtil.cachedResult(
+  () => internalUtil.filterDuplicateStrings(binding.getSSLCiphers(), true)
+);
```

Comment:
This reformatting is being done so it matches the formatting in `lib/crypto.js` so all four functions being talked about here are the same. I can take it out if it's seen as extraneous churn for this change. Internally, I'm +0 on including it. /cc @sam-github who flagged this in another PR